src/public/modules/forms/helpers/decryption.worker.js

```diff
@@ -84,13 +85,51 @@ function verifySignature(decryptedSubmission, created) {
   return verified.every((v) => v)
 }
 
-/** @typedef {{
- *    created: string,
- *    id: string,
- *    status?: string
- *    submissionData?: Object
- * }}
- * CsvRecord */
+class CsvRecord {
+  constructor(id, created, status) {
+    this.id = id
+    this.created = created
+    this.status = status
+
+    this._statusMessage = status
+    this._record = []
+
```

Comment:
I realise from the implementation this is implicit
```suggestion
this.submissionData = undefined
```